service/javadsl/server/src/main/java/com/lightbend/lagom/javadsl/server/ServiceGuiceSupport.java

```diff
@@ -3,23 +3,43 @@
  */
 package com.lightbend.lagom.javadsl.server;
 
-import com.lightbend.lagom.internal.javadsl.*;
+import com.google.inject.Binder;
+import com.lightbend.lagom.internal.javadsl.BinderAccessor;
+import com.lightbend.lagom.internal.javadsl.server.JavadslServicesRouter;
 import com.lightbend.lagom.internal.javadsl.server.ResolvedServices;
 import com.lightbend.lagom.internal.javadsl.server.ResolvedServicesProvider;
 import com.lightbend.lagom.internal.javadsl.server.ServiceInfoProvider;
-import com.lightbend.lagom.internal.javadsl.server.JavadslServicesRouter;
 import com.lightbend.lagom.internal.server.status.MetricsServiceImpl;
-
-import com.lightbend.lagom.javadsl.server.status.MetricsService;
-import com.google.inject.Binder;
 import com.lightbend.lagom.javadsl.api.ServiceInfo;
 import com.lightbend.lagom.javadsl.client.ServiceClientGuiceSupport;
+import com.lightbend.lagom.javadsl.server.status.MetricsService;
+
+import java.util.Arrays;
 
 public interface ServiceGuiceSupport extends ServiceClientGuiceSupport {
 
+    // redefines bindServiceInfo from super interface.
```

Comment:
Better to have proper javadocs here, including mentioning that this should only be used when there's no services to bind, and saying that one of `bindServiceInfo` or `bindServices` must be invoked, but not both.